### PR TITLE
RVAT_720_722_VerifyExistenceOfDropdownSubMenuEvents

### PR DESCRIPTION
**Pull request by MelnykNikita**

### CODE REVIEW

File: src/test/java/filterEventsOnTheMapByCategories/US_FilterEventsOnTheMapByCategoriesWithChrome.java [hunk 1]
```diff
@@ -1,8 +1,73 @@
...
+        driver = new ChromeDriver();
+        driver.manage().window().maximize();
+        wait = new WebDriverWait(driver,MAX_WAIT_TIME);
+        initializeLocators();
+        setUpPreconditions();
+    }
+
+    @AfterMethod
+    public void quitDriver() throws InterruptedException {
+        Thread.sleep(2000);
```
Comment (TyshchenkoRoma) [inline src/test/java/filterEventsOnTheMapByCategories/US_FilterEventsOnTheMapByCategoriesWithChrome.java]: I think that it is extra line.

File: src/test/java/filterEventsOnTheMapByCategories/US_FilterEventsOnTheMapByCategoriesWithChrome.java [hunk 2]
```diff
@@ -1,8 +1,73 @@
...
+    }
+
+    @Test
+    public void verifyExistenceOfEventsDropdownList(){
+        Assert.assertTrue(exists(eventsDropdownList), "This element doesn't exist");
+    }
+
+    public void setUpPreconditions(){
+        driver.get(mapLink);
+        categoriesDropdownList = wait.until(ExpectedConditions.presenceOfElementLocated(categoriesLocator));
```
Comment (TyshchenkoRoma) [inline src/test/java/filterEventsOnTheMapByCategories/US_FilterEventsOnTheMapByCategoriesWithChrome.java:165]: You can add ".click" at the end of this line.

**Review by TyshchenkoRoma — APPROVED** [review]

File: src/test/java/filterEventsOnTheMapByCategories/US_FilterEventsOnTheMapByCategoriesWithChrome.java (hunk 1 above)
Comment (dmatsev) [reply src/test/java/filterEventsOnTheMapByCategories/US_FilterEventsOnTheMapByCategoriesWithChrome.java]: +1

File: src/test/java/filterEventsOnTheMapByCategories/US_FilterEventsOnTheMapByCategoriesWithChrome.java [hunk 3]
```diff
@@ -1,8 +1,73 @@
...
+        eventsDropdownList = wait.until(ExpectedConditions.presenceOfElementLocated(eventsLocator));
+        eventsDropdownList.click();
+    }
+
+    public void initializeLocators(){
+        categoriesLocator = By.xpath("//*[@id=\"main\"]/div/place-list/div/div/div[1]/div[2]/div/div[3]");
+        eventsLocator = By.xpath("//*[@id=\"main\"]/div/place-list/div/div/div[1]/div[2]/div/div[3]/ul/li[3]");
+    }
+    public boolean exists(WebElement element){
+        if (element == null){
```
Comment (dmatsev) [inline src/test/java/filterEventsOnTheMapByCategories/US_FilterEventsOnTheMapByCategoriesWithChrome.java]: I don't understand this

**Review by dmatsev — CHANGES_REQUESTED** [review]
The same comments are valid for Firefox

**Review by MaxKovalets — APPROVED** [review]

**Pull request merged**